funcbench/manifests/benchmark/2_job.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: workflow
+  namespace: funcbench-{{ .PR_NUMBER }}
+data:
+  event.json: |
+    {
+      "action": "created",
+      "issue": {
+        "number": {{ .PR_NUMBER }}
+      },
+      "repository": {
+        "name": "{{ .GITHUB_REPO }}",
+        "owner": {
+          "login": "{{ .GITHUB_ORG }}"
+        }
+      }
+    }
+  BRANCH: {{ .BRANCH }}
+  RACE: {{ .RACE }}
+  REGEX: {{ .REGEX }}
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: funcbench-test-{{ .PR_NUMBER }}
+  namespace: funcbench-{{ .PR_NUMBER }}
+spec:
+  # never re-create a new pod
+  backoffLimit: 0
+  template:
+    spec:
+      restartPolicy: Never
+      initContainers:
+      - name: clean-up
+        image: busybox:latest
+        command: ['sh', '-c', 'rm -fr /build/* && mkdir /build/tmp && mkdir /build/src']
+        volumeMounts:
+        - name: instance-ssd
+          mountPath: /build
+      containers:
+      - name: funcbench
+        image: nevill/funcbench:latest
+        imagePullPolicy: Always
+        env:
+          - name: GITHUB_WORKSPACE
+            value: /build/src
+          - name: GITHUB_TOKEN
+            value: {{ .GITHUB_TOKEN }}
+          - name: GITHUB_SHA
+            value: {{ .LAST_COMMIT_SHA }}
+          - name: GOTMPDIR
+            value: /build/tmp
+          - name: TMPDIR
```

Comment:
why do we need these 2 env vars? GOTMPDIR and TMPDIR

can we use the default linux tmp dir for all these?

also why do we need GITHUB_WORKSPACE?

Reply:
These 3 variables `GITHUB_WORKSPACE`, `GOTMPDIR`, `TMP_DIR` are used for generated testing binary, to make sure tests are run attached local SSD disk. But it can be replaced by specify disktype to `pd-ssd` when creating a VM. These will be removed later.

Reply:
why not just use the linux tmp dir?
/tmp

Reply:
because `/tmp` is not on a SSD disk.

Reply:
in this case this doesn't matter as there is no requirement for high io